templates/themes/peak/pagelayout/header/login.html.twig

```diff
@@ -37,15 +37,15 @@
                 <span class="user-icon user-icon--main"></span>
             {% endif %}
 
-            <a class="btn btn-user" href={{ path('backoffice_dashboard_index') }}>{{ 'layout.my_dashboard'|trans }}</a>
-            <a class="btn btn-user" href={{ path("backoffice_my_account_personal_details") }}>{{ 'layout.my_profile'|trans }}</a>
+            <a class="btn btn-user" href={{ path('backoffice_dashboard_index') }}>{{ 'ngsite.layout.my_dashboard'|trans }}</a>
```

Comment:
We will need to address the modifications in this file too, since this file is also generic and used for F&H site, so this needs to be moved to peak design, but I will take a look at it a bit later.

I will merge this as is for now.